jerry-core/ecma/base/ecma-helpers-values-collection.cpp

```diff
@@ -38,7 +38,7 @@ ecma_new_values_collection (const ecma_value_t values_buffer[], /**< ecma-values
 {
   JERRY_ASSERT (values_buffer != NULL || values_number == 0);
 
-  const size_t values_in_chunk = sizeof (ecma_collection_chunk_t::data) / sizeof (ecma_value_t);
+  const size_t values_in_chunk = sizeof (((ecma_collection_chunk_t *)0)->data) / sizeof (ecma_value_t);
```

Comment:
@rtakacs I don't think it's a hack. it is a well accepted method to get size of c struct member. (see http://stackoverflow.com/questions/3553296/c-sizeof-single-struct-member and http://stackoverflow.com/questions/3864583/sizeof-a-struct-member)

your solution directly calculated the size of `ecma_collection_chunk_t::data`